core-bundle/src/Twig/Extension/ContaoExtension.php

```diff
@@ -394,15 +394,11 @@ public function addDocumentContent(string|null $identifier, string $content, Doc
         // TODO: This should make use of the response context in the future.
         if (DocumentLocation::head === $location) {
             if (null !== $identifier) {
-                $GLOBALS['TL_HEAD'][$identifier] = $content;
+                $GLOBALS['TL_TWIG_CSS'][$identifier] = $content;
```

Comment:
Things you put in `DocumentLocation::head` can be anything, not just styles.

Reply:
Okay, then you do prefer another target?

Reply:
Yes, either that or a priority of some sorts. Because now it would get confusing if you're for instance adding a meta tag or a script.

By the way: I think we should not have Twig in the name, because things could also be added to the response context outside of templates.

Maybe we should discuss what's the best solution in the upcoming call? The changes have some implications on how people should write their templates and I would like to get it right in the first place ☺️. For example, for bundles something like sorting in the loading order could be the right choice.

Reply:
Can't we simply output the user stylesheets last in the template? I.e. after `TL_HEAD`?

Reply:
No, because people might have a `<style>` tag in `TL_HEAD`. We must not change the existing order, otherwise it would be a BC break.